.golangci.json

```diff
@@ -3,7 +3,7 @@
 		"disable-all": true,
 		"enable": [
 			"govet",
-			"golint",
+			"revive",
```

Comment:
Is there a reason for using revive over goling?

Reply:
no longer officially supported, revive is the spiritual successor to that, at least according to golangcilint